src/Youtube.js

```diff
@@ -33,7 +33,7 @@ THE SOFTWARE. */
 }(this, function(videojs) {
   'use strict';
 
-  var _isOnMobile = videojs.browser.IS_IOS || videojs.browser.IS_ANDROID;
+  var _isOnMobile = videojs.browser.IS_IOS || useNativeControlsOnAndroid();
```

Comment:
would videojs.browser.IS_NATIVE_ANDROID be sufficient?
https://github.com/videojs/video.js/blob/94e54fccf854786e01bd302bc18d93577ee4e25b/src/js/utils/browser.js#L58